src/app/components/navbar/navbar.component.html

```diff
@@ -24,13 +24,32 @@
     </ng-container>
 
     <div class="toolbar-spacer"></div>
+
     <div [hidden]="(currentExperiment|async) == null">
         <span>{{currentExperiment|async}}</span>
         <a class="navigation-link" mat-icon-button [hidden]="(currentExperiment|async) == null" [routerLink]="['/']"
             aria-label="change experiment">
             <mat-icon>swap_horiz</mat-icon>
         </a>
     </div>
+    <button class="downloads-button" mat-button [matMenuTriggerFor]="downloadList">
+        <mat-icon [matBadge]="downloadBadgeCounter | async" [matBadgeHidden]="!(downloadBadgeCounter | async)"
+            matBadgeColor="warn" matBadgeSize="small" (click)="resetBadge()">download</mat-icon>
```

Comment:
The badge count should always correspond to the number of available (and or preparing) downloads. It should not reset on every view but by actively removing downloads from the list.